cli/tests/testdata/worker_drop_handle_race_terminate.js

```diff
@@ -16,8 +16,9 @@ const WORKER2 = getCodeBlobUrl(`
   // We sleep for slightly under 2 seconds in order to make sure that worker 1
   // has closed, and that this worker's thread finishes normally rather than
   // being killed (which happens 2 seconds after calling terminate).
-  Deno.sleepSync(1800);
```

Comment:
I think you might have to use `Atomics.wait` for some of these and using `setTimeout` might possibly make some tests flakier.

Reply:
CC @andreubotella

Reply:
The synchronous sleep here is intentional (the thread should be blocked, and the event loop should not be executing). As such you can not just replace this with `setTimeout`.